src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+
+  &__card {
+    width: 200px;
+    margin-bottom: 46px;
```

Comment:
what about left out bottom margins on the last row?
![image](https://user-images.githubusercontent.com/37161315/216421114-ebd51956-7666-45c9-881e-e4a22191b21e.png)